client/src/main/views/pointingRoom/components/mainContent/Participant.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+const getInitials = (name) => {
```

Comment:
Move this function to a new file named helperFunctions inside of client/src/main/utils

Export it there and import it here.